frontend/src/prompted_readings/student_view.js

```diff
@@ -278,10 +398,22 @@ class Study extends React.Component {
     }
 
     continueButtonClick() {
+        /*
+         * Handles what to do when the user advances beyond the page that checks if they wish to go back
+         */
+
         this.setState({show_go_back: false, show_response:true});
     }
 
     handleStoryScroll(e) {
+        /*
+         * For large enough stories, this function logs how many times the user scrolls up to read a previous portion
```

Comment:
Look at maximum line width -- are you getting warnings there or is this okay?

Reply:
PyCharm thinks the code is still within the character limit, but I'll space it out more just in case.

Reply:
If it's fine, don't worry about it!  :-)  I just wanted to check.

Reply:
I'm going to add a ```.editorconfig``` file that should resolve this, limiting the line-length to 100.